draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -416,7 +416,10 @@
                 offerer MUST insert an SDP 'dtls-id' attribute with the previously assigned value in the offer.
                 In addition, the offerer MUST insert an SDP 'setup' attribute with a value that does
                 not change the previously negotiated DTLS roles, and one or more SDP 'fingerprint' attributes with
-                values that do not change the previously sent fingerprints, in the offer.
+                values that do not change the previously sent fingerprints, in the offer. The value of the 'setup'
+                attribute SHOULD be set to 'actpass', in order to allow the answerer to establish a new DTLS
+                association with a different role, but MAY be set to the current negotiated role ('active' or
+                'passive').

```

Comment:
This sounds contradictory for me.

1. What I interpreted from "not change the previously negotiated DTLS roles" is that "active" or "passive" should be used, which are DTLS roles ("actpass" is not a role, I think). Anyway, the author of this text could  clarify this.
2. The offerer aims to not establish a new DTLS association, but allows the answerer to do it.